docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -1,213 +1,185 @@
 # BigDL-LLM Installation: GPU
 
+## Windows
 
-## PyTorch 2.1
-### Prerequisite
+### Prerequisites
 
-```eval_rst
-.. tabs::
-   .. tab:: Linux
-
-      BigDL-LLM for GPU supports on Linux with PyTorch 2.1 has been verified on:
+BigDL-LLM on Windows supports Intel iGPU and dGPU.
 
-      * Intel Arc™ A-Series Graphics
-      * Intel Data Center GPU Flex Series
-      * Intel Data Center GPU Max Series
+To apply Intel GPU acceleration, there're several prerequisite steps for tools installation and environment preparation:
 
-      .. note::
+* Step 1: Install [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) Community Edition and select "Desktop development with C++" workload
```

Comment:
It would be better to provide a download screenshot here.

Reply:
We could use the official screenshot here: https://learn.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-170#step-4---choose-workloads

Not sure about the license problem